### ISSUE TITLE
Instantiation of NotionClient fails

### ISSUE THREAD

**Issue by weaversam8** [jamalex/notion-py#25]
After trying to follow the basic tutorial in the README, I encountered an error.

```
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\notion\store.py", line 111, in _load_cache
    with open(self._get_cache_path(attr)) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\username\\.notion-py\\cache\\edf90714b95fc442ac97965a8e2fb37795a647ed342bd73df657bcd3c6a016ea_values.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".\index.py", line 3, in <module>
    client = NotionClient(token_v2='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx')
  File "C:\Python34\lib\site-packages\notion\client.py", line 33, in __init__
    self._store = RecordStore(self, cache_key=cache_key)
  File "C:\Python34\lib\site-packages\notion\store.py", line 82, in __init__
    self._load_cache()
  File "C:\Python34\lib\site-packages\notion\store.py", line 117, in _load_cache
    except (FileNotFoundError, json.JSONDecodeError):
AttributeError: 'module' object has no attribute 'JSONDecodeError'
```

Any ideas as to what could be causing this?

**Comment by weaversam8** [comment]
After investigating the code, it looks like the error is being surfaced by [line 117 in `store.py`](https://github.com/jamalex/notion-py/blob/573cf67a4ab7ce10558c6a666b7c77b5728de93c/notion/store.py#L117). A `FileNotFoundError` is being generated by line 111, and then it is entering the `except` statement. Inside the `except` statement we're also noting that we could catch a `JSONDecodeError` from the `json` package, but for some reason that isn't an attribute in the package?

**Comment by weaversam8** [comment]
Aha! It seems that in Python 3.4 a simple `ValueError` is raised on a JSON decode failure. The `JSONDecodeError` attribute isn't added until Python 3.5!

Source: [https://stackoverflow.com/a/44714607/4196127](https://stackoverflow.com/a/44714607/4196127)

This package is described as an "Unofficial Python 3 client for Notion.so API v3." I'd suggest an to `README.md` to indicate it requires Python 3.5 at minimum.

**Comment by weaversam8** [comment]
I've just made the above pull request to add a note about the minimum version required.

**Comment by jamalex** [comment]
Thanks for the catch! I had tested on 3.4, but I guess not with that file missing, so it didn't trigger it. Since `JSONDecodeError` is a subclass of `ValueError`, I was able to just use `ValueError`:
https://github.com/jamalex/notion-py/commit/48c0fd686ddb1055861f973653d73f88cd82b117

This will be released in the upcoming 0.0.22.
